packages/components/.storybook/main.cjs

```diff
@@ -19,7 +19,11 @@ module.exports = {
     './assets',
     {
       from: '../../placeholders/src/images',
-      to: '/placeholders'
+      to: '/placeholders/images'
+    },
+    {
+      from: '../../placeholders/src/videos',
+      to: '/placeholders/videos'
```

Comment:
This is not a breaking change, as the original paths are unchanged.

Reply:
good point but looking at the changes below, users would need to change the path as we had to in the stories... Or what to we both miss... ;-)

Reply:
?

"from" didn't change and there was just added a new video. This feature has nothing to do with Storybook or how others configure it.

There is 0 change for people using the placeholder package.

Reply:
yeah, the from is the same but the to path is now different for images. Thats why I though it might be a Breaking Change. 
So if someone is using our placeholder images and we changing the to path.. would that not break their application?

Reply:
The package didn't change the path. It's still `/images`. We added in OUR components docs `/videos`, but that has nothing to do how others use the package.

Reply:
In that case it's especially harmful, as Semantic Release would even increase do a major release for the components package. (A readon, why we should think about separating docs from components in future AND maybe using `changesets`.)

Reply:
Allright, I removed the description.

Reply:
my bad... mixed up the files and overlooked, that we are talking about storybook config. Sorry for that.